lib/rules/container-name-pattern/__tests__/index.mjs

```diff
@@ -0,0 +1,133 @@
+import rule from '../index.mjs';
+import { stripIndent } from 'common-tags';
+
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: ['foo-.+'],
+
+	accept: [
+		{ code: '@container foo-bar {}' },
+		{ code: ' @CONTAINER foo-bar   {}' },
+		{ code: ' @container foo-bar {}' },
+		{ code: '@container foo-bar {\n}' },
+		{ code: '@container foo-bar (width > 400px) {}' },
+		{ code: '@container foo-bar () {}' },
+		{ code: '@container foo-bar style(--cards small) {}' },
+		{ code: '@container foo-bar scroll-state(stuck: top) {}' },
+		{ code: '@container foo-bar (inline-size > 30em), foo-baz style(--large: true) {}' },
+		{ code: '@container foo-bar (inline-size > 30em) and foo-baz style(--large: true) {}' },
+		{
+			code: '@container () {}',
+			description: 'nameless container condition',
+		},
+
+		{
+			code: stripIndent`
+				.my-component {
+					container-name: foo-bar;
+				}
+			`,
```

Comment:
```suggestion
			code: 'a { container-name: foo-bar; }'
```

Reply:
Let's update the tests to match the [conventions](https://stylelint.io/developer-guide/rules#write-tests).